app/javascript/packs/personal-key-page-controller.js

```diff
@@ -69,16 +24,16 @@ function downloadForIE(event) {
   window.navigator.msSaveBlob(blob, filename);
 }
 
+function trackAcknowledgment(clickEvent) {
+  trackEvent('IdV: personal key acknowledgment toggle', { checked: clickEvent.target.checked });
```

Comment:
I saw this had changed from "toggled" to "toggle" in a commit. Was it intentional? Most existing events use the past tense, so I'd expect "toggled"

```suggestion
  trackEvent('IdV: personal key acknowledgment toggled', { checked: clickEvent.target.checked });
```

Reply:
> Lint insisted.

Which lint? Based on [the build failure prior to that commit](https://gitlab.login.gov/lg/identity-idp/-/jobs/98577), the lint appears to only be about the whitespace, not the name of the event.